trollflow2/launcher.py

```diff
@@ -319,24 +328,27 @@ def get_dask_client(config):
 
 def queue_logged_process(msg, prod_list, produced_files, log_queue):
     """Run `process` with a queued log."""
-    _reset_log_handlers()
-    handler = handlers.QueueHandler(log_queue)
-    LOG.addHandler(handler)
+    setup_queued_logging(log_queue)
+    with suppress(ValueError):
+        signal.signal(signal.SIGUSR1, print_traces)
+        LOG.debug("Use SIGUSR1 on pid {} to check the current tracebacks of this subprocess.".format(os.getpid()))
```

Comment:
Does this work only for subprocesses or also for threads?

Reply:
Only for subprocesses unfortunately.